### ISSUE TITLE
Buttons missing type="button"

### ISSUE THREAD

**Issue by afercia** [voldemortensen/menu-customizer#109]
Checking the plugin, the following buttons miss a `type="button"` attribute:
```
<button class="customize-panel-back" tabindex="-1">
<button class="customize-screen-options-toggle" tabindex="0" aria-expanded="false">
<button class="customize-help-toggle dashicons dashicons-editor-help" tabindex="0" aria-expanded="false">
<button class="button-secondary add-new-menu-item add-menu-toggle" tabindex="0">
```

**Comment by afercia** [comment]
The previous 4 ones are all in `menu-customize-controls.php` and there's also a fifth one in `WP_Customize_Menus::available_items_template()`
```<button class="customize-section-back" tabindex="-1">```
